guide/how-it-works/silent-payments.md

```diff
@@ -0,0 +1,400 @@
+Below is a summary of the keys & addresses i
+
+| Silent payments' concept       | Shared with         | for: |
+|:-------------|:------------------|:------|
+| Static address | Senders | Making payments
+| Scan key     | Nodes & [light clients](/guide/how-it-works/nodes/#light-nodes)  | Payment scanning information
+| Spend key   | Wallet owner | Spending bitcoin
+
+
+The above model introduces a scan key that is used for payment scanning and a static address which is used for deriving on-chain addresses. This is in contrast with the BIP-32 model where [extended public keys](/guide/glossary/#extended-public-key-xpub-ypub-zpub) are used for both purposes. Below is a summary of the differences between the two models:
+
+| Current model with BIP-32       | Silent payments model          | Benefit  |
+|:-------------|:------------------|:------|
+| Single-use on-chain address           | Static address | Reusable & untraceable on-chain |
+| Extended private key (xprv) | Spend (private) key   |   |
+| Extended public key (xpub)           | Scan (public) key      | Nodes & [light clients](/guide/how-it-works/nodes/#light-nodes) can scan for payments but cannot derive addresses    |
+| *Address-based model*           | *Contact-based model* | *More intuitive and familiar*  |
+
+
+## Labels & contacts
+
+### Labels
+
+<div class="center" markdown="1">
+
+{% include picture.html
+   image = "/assets/images/guide/how-it-works/silent-payments/adding-labels-to-static-address.png"
+   retina = "/assets/images/guide/how-it-works/silent-payments/adding-labels-to-static-address@2x.png"
+   modalImage = "/assets/images/guide/how-it-works/wallet-privacy/network-settings-big.png"
+   alt-text = "An application screen showing labels being added to a static address."
+   caption = "Adding labels to a static address before posting or sharing with others."
+   width = 250
+   height = 542
+   layout = "float-right-desktop -background -shadow"
+%}
+
+Labels or tags (such as 'business' or 'no-kyc') are brief identifiers often associated with information such as payments, transactions and addresses. They're used to organise, categorise or quickly spot items from a large set.
+
+Silent payments allows users to add labels to their static addresses on a case-by-case basis before sharing or posting them. When a payment is received, these labels can be detected and used by the receiver (but no one else) to identify the static address used to make the payment.
+
+</div>
+
+For example, Alice can add can customise her static address with different labels before sharing them
+- with a certain client
+- on nostr and
+- on her website
+
+When she receives payments from any of these sources, the respective label(s) will be detected in the on-chain address and used to infer the source of the payment.
+
+{% include picture.html
+   image = "/assets/images/guide/how-it-works/silent-payments/how-silent-payment-labels-work.png"
```

Comment:
Added desktop modals in https://github.com/BitcoinDesign/Guide/pull/1109/commits/55f60e332247a02c15dd043db7abcd8ff475e483. Mobile modals are not required since I adjusted their aspect ratio to approximate the avg mobile screen aspect ratio  so they already show full size without the modal.